### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by gdvl777**
You should implement the following interactions:

- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.

In order to associate each project with the popup window details, you must refactor the project section:

- You need to use a JavaScript array to store all of the information for all projects.
- For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
- You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
- After that, you could implement the popup window.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+            <span></span>
+            <h6 class="year"></h6>
+          </div>
+          <img src="#" alt="img" class="modal-img">
+          <div class="project-desc">
+            <p class="desc">Lorem ipsum dolor sit amet consectetur adipisicing elit. In voluptatem pariatur modi rerum error
+              ea consequatur, excepturi eum maiores! Quo quibusdam culpa ratione.?Lorem ipsum dolor sit amet, consectetur
+              adipisicing elit.
+              !</p>
+            <div class="desc_btn">
```
Comment (brainconnect93) [inline src/index.html]: - [x] Hello, great effort working on your project, but as requested in the project, you should create the pop up also dynamically and the hard coded line of code in your HTML file should be removed, just the way you have done for the project cards. Kindly remove it, only then we can be sure your pop up card is created dynamically. 👍 👍

File: src/styles.css [hunk 2]
```diff
@@ -0,0 +1,1198 @@
...
+  width: 65%;
+}
+
+.modal-example .closeModal {
+  font-size: 50px;
+  color: #67798e;
+  border: none;
+  outline: none;
+  background: none;
+}
```
Comment (brainconnect93) [inline src/styles.css]: - [x] Great effort working on your modal pop up cards, but currently what you have in your design is quite different from what we have in the Figma design. and i would want you to pay optimum attention to the Figma design to have a wonderful replica of the design in your work. Below is a video recording highligting some of the changes you will have to pay attention to. 👇  

- [x] Also i notice while working on making your project cards dynamic the design has been disrupted on both the `mobile view` and `desktop view`. I would kindly want you to revisit that, as though it is not part of this milestone but it is very paramount to have it fixed, for you to have a wonderful portfolio project to showcase.

  Video 👇 

https://github.com/gdvl777/Portfolio-finish-mobile-version/assets/78634841/2b4b5bc4-39eb-4fdd-ae7b-1c45df46b09d

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @gdvl777, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

-  Descriptive Pull Request title and summary. ✔️
- User clicks the button, the popup with details about the project appears ✔️
- User clicks the close (X) button, the popup disappears ✔️
- Modal popup data is populated with the data from the projects array ✔️
- Good use of Github flow ✔️

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by descholar-ceo — APPROVED** [review]
Hi @gdvl777 🤝  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Pull Request has a title and a description. ✔️
👍🏽 The user clicks the button, and the popup with details about the project appears ✔️
👍🏽 The user clicks the close button, and the popup disappears ✔️
👍🏽 Modal popup data is populated with the data from the projects array ✔️
👍🏽 GitHub flow used correctly ✔️
👍🏽 Commit message and the readme file are professional ✔️ 
👍🏽 The PR passes the linters check ✔️ 
👍🏽 The comments pointed out in the previous review, has been addressed successfully ✔️ 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by frank1738 — COMMENTED** [review]
Hi @gdvl777 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] You have submitted a project that has already been approved. If you are working on a different milestone, Kindly create another pull request

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request closed without merging**